### PR TITLE
Log all the things

### PR DESCRIPTION
**Pull request by roosnic1**

### CODE REVIEW

File: packages/api/src/server.ts [hunk 1]
```diff
@@ -40,21 +51,22 @@ export class WepublishServer {
...
               }
             })
           } else {
-            console.warn(`${methodName} does not exist in dbAdapter[${mtp.key}]`)
+            logger('server').warn('%s does not exist in dbAdapter[%s]', methodName, mtp.key)
           }
         })
       } else {
-        console.warn(`${mtp.key} does not exist in dbAdapter`)
+        logger('server').warn('%s does not exist in dbAdapter')
```
Comment (penk85) [inline packages/api/src/server.ts:69]: Should this have the variable? `('%s does not exist in dbAdapter', mtp.key)`

**Review by penk85 — APPROVED** [review]

**Review by bishoymelek — APPROVED** [review]
:+1: :tada:

**Review by bishoymelek — APPROVED** [review]
Great started using it already in the cron jobs 👌

**Comment by faebebin** [comment]
Will review tomorrow

File: packages/api/src/server.ts [hunk 2]
```diff
@@ -102,6 +121,11 @@ export class WepublishServer {
       cors: corsOptions
     })
 
+    app.use((err: any, req: Express.Request, res: Express.Response, next: NextFunction) => {
```
Comment (0815pascal) [inline packages/api/src/server.ts:124]: Not sure if I misunderstand something here but where do you use `req` and `res` in this function? I only see `err` being used

**Review by 0815pascal — CHANGES_REQUESTED** [review]

File: packages/api/src/server.ts (hunk 2 above)
Comment (roosnic1) [reply packages/api/src/server.ts:124]: because `next` is also used

**Review by faebebin — APPROVED** [review]
Looks good.

**Pull request merged**